cli/lib/args.go

```diff
@@ -75,13 +75,13 @@ func ParseLibraryReferenceArgs(args []string) ([]*LibraryReferenceArg, error) {
 // ParseLibraryReferenceArgAndAdjustCase parse a command line argument that reference a
 // library and possibly adjust the case of the name to match a library in the index
 func ParseLibraryReferenceArgAndAdjustCase(instance *rpc.Instance, arg string) (*LibraryReferenceArg, error) {
-	libRef, err := ParseLibraryReferenceArg(arg)
+	libRef, _ := ParseLibraryReferenceArg(arg)
```

Comment:
Shouldn't we fix this by handling this `err` instead of ignoring it?

Reply:
Probably already clear, but I did this because this assignment of `err` was ignored and the `error` type from the declaration was incompatible with the change of `lib.LibrarySearch()`'s return type from `error` to `*status.Status`.

If it's to be ignored, I think it's best to make that explicit by using `_`. But if an error return from `ParseLibraryReferenceArg()` is significant in this context, then of course it's better to handle it.

Reply:
Yep, the `err` was ignored also before this PR (it was overwritten without being checked). This should be fixed in another PR BTW.